README.md

```diff
@@ -1698,6 +1698,7 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 Software for easy installation, management and configuration of self-hosted services and applications.
 
 - [Ansible-NAS](https://github.com/DaveStephens/ansible-nas) - Build a full-featured home server with this playbook and an Ubuntu box. `MIT` `YAML/Docker`
+- [Aetolos](https://gitlab.com/noumenia/aetolos) - Aetolos is a secure open source control panel for virtual hosting. `AGPL-3.0` `PHP`
```

Comment:
If this is a control panel it must be added here instead: https://github.com/awesome-foss/awesome-sysadmin#control-panels

@CrimsonTome could you resubmit a pull request there?

Reply:
Sorry for the late response, will resubmit this PR now.